packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -70,34 +72,37 @@ copyConfigFile() {
 templateFilePath="${0%/*}/../templates/function-app-template.json"
 
 # Read script arguments
-while getopts "r:e:k:d" option; do
+while getopts "r:c:e:k:d:" option; do
     case $option in
     r) resourceGroupName=${OPTARG} ;;
+    c) clientId=${OPTARG} ;;
     e) environment=${OPTARG} ;;
     k) keyVault=${OPTARG} ;;
     d) dropFolder=${OPTARG} ;;
     *) exitWithUsageInfo ;;
     esac
 done
 
-if [ -z $resourceGroupName ] || [ -z $environment ] || [ -z $keyVault ]; then
+if [ -z $resourceGroupName ] || [ -z $clientId ] || [ -z $environment ] || [ -z $keyVault ]; then
     exitWithUsageInfo
```

Comment:
you shouldnt fail if clientId is not set. you should only fail if client id is not set & environment is not dev.

Reply:
I'll fix it in another PR